macaw/core/nlp_pipeline/nlp_pipeline.py

```diff
@@ -0,0 +1,19 @@
+from typing import List
+
+from core.interaction_handler import Message
+
+
+class NlpPipeline:
+    def __init__(self, modules: dict):
+        self.modules = dict()
+        self.modules.update(modules)
+        self.context = dict()
+
+    def run(self, conv_list: List[Message]):
```

Comment:
This probably should be parallelized. You could probably do something similar to `RequestDispatcher`

Reply:
Yeah. That is better. I'll do this in another commit where I have written some actual NLP modules. That will also help in testing. In this PR it's just an empty method. Sometime later we could make it a DAG just like Cobot had it.